fiatconnect-api.md

```diff
@@ -1915,21 +1915,22 @@ Amounts of fiat currencies MUST be given in the units of that currency, e.g. `5.
 
 Given that amounts are communicated in divisible units-- such as US Dollars, which are divisible into `100` cents, or Celo, which 
 is divisible into `10^18` wei--  it is expected that the client or server will sometimes need to communicate non-integer amounts. 
-For all amounts, [decimal notation](https://en.wikipedia.org/wiki/Decimal#Decimal_notation) MUST be used-- for example, 
-One half should be given as `0.5`.
+For all amounts, [decimal notation](https://en.wikipedia.org/wiki/Decimal#Decimal_notation) MUST be used, and the decimal 
+separator MUST be the period symbol: `.` . For example, one half should be given as `0.5`. 
+
+Put another way, all amounts MUST match the following regex: `/^[0-9]+\.?[0-9]*$/` .
 
 ## 6.3 Precision
 
-FiatConnect-compliant API's MUST observe 18 decimal places of precision for all cryptocurrencies and 2 decimal places of
-precision for all fiat currencies. 
+FiatConnect-compliant API's MUST observe the maximum number of decimal places of precision for every cryptocurrency in `CryptoTypeEnum`, and 2 decimal places of
+precision for all fiat currencies. (Most, but not all, ERC-20 tokens offer up to 18 decimal places of precision.)
```

Comment:
super-nit: ERC20 tokens can also specify [more than 18 decimals](https://ethereum.stackexchange.com/questions/118896/can-an-erc-20-have-more-than-18-decimals#:~:text=Theoretically%2C%20you%20could%20use%20any,tokens%20in%20terms%20of%20supply. of precision

Reply:
it's still true that most offer up to 18 decimals

Reply:
for example, the Celo token offers up to 18 decimal places of precision